src/snowflake/snowpark/_internal/lazy_import_utils.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
+#
+
+import importlib
+from typing import Any
+
+
+def lazy_import(module_name: str) -> Any:
```

Comment:
nit: I don't think we need to wrap one single line function call into another function.
if you prefer the name `lazy_import` you can do
```python
lazy_import = importlib.import_module
```